services/app/src/helpers/openWsConnection.js

```diff
@@ -0,0 +1,6 @@
+import wsWrapper from "./wsWrapper";
+
+export const openWsConnection = (url) => {
```

Comment:
Are you planning on reusing this?
If no, move it into the ws-context file as a separate function.